ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_trackings.schema.yml

```diff
@@ -7,15 +7,19 @@
 type: dict
 keys:
   flow_trackings:
+    deprecation:
+      warning: true
+      new_key: flow_tracking
+      remove_in_version: v5.0.0
     type: list
     primary_key: type
     items:
       type: dict
       keys:
         type:
           type: str
-          valid_values: ['sampled']
-          description: Flow Tracking Type - only 'sampled' supported for now
+          valid_values: ['sampled', 'hardware']
```

Comment:
We should revert these changes, and only "sampled" should be supported in the old model.

Reply:
resolved in [713d986](https://github.com/aristanetworks/ansible-avd/pull/3199/commits/713d986fbe61d087620d72d5591bc7d02ca88c2f)